syntax = "proto2";

message s2u_player_heartbeat_request {
	required uint64 player_id = 1;
}